react/modules/tasks/nextjs-ssr-ssg.md

```diff
@@ -0,0 +1,41 @@
+# Next.js. Server Side Rendering
+
+## Technical requirements
+
+1. Create a separate branch for this task from the previous branch task. Branch name: "nextjs-ssr-pages-api".
+2. Next.js Pages API Integration:
+   - Migrate your application from vite to next.js by using Pages folder API. You'll need to adapt existing libraries to work with next.js.
+3. Next.js App Router API Integration:
+   - Create a separate branch for this point from "nextjs-ssr-pages-api" branch. Branch name: "nextjs-ssr-app-router-api".
+   - Migrate from Pages Api to App Folder Api.
+4. Remix Integration:
```

Comment:
Same for Remix, although it is built on top of React Router, I'm not sure that we need to specify it explicitly.

Reply:
Didn't catch your comment. Could your explain please?

Reply:
Me neither :) I guess this comment was about routing, and can be considered as a duplicate. Resolving it.